STYLEGUIDE.md

```diff
@@ -37,7 +41,7 @@ foo(f1,
     f2)
 ```
 
-## Imports
+### Imports
```

Comment:
@renancaraujo you added a lot of things but didn't fix the one TODO we had:

```
TODO Do/Do not use the imports grouping internally (like component.dart, extensions.dart, etc).
```

Reply:
done